src/phoenix/server/main.py

```diff
@@ -182,34 +184,50 @@ def _get_pid_file() -> Path:
             "database is new."
         ),
     )
+    serve_parser.add_argument(
+        "--scaffold-dataset",
```

Comment:
can we make this plural and support multiple?

Reply:
Yes, this should be plural and already supports multiple datasets. 

I did want to ask though if we wanted to be able to scaffold the datasets independently of fixtures. Currently, the flag doesn't take in any inputs and instead enables the ingestion of datasets defined under trace fixtures that are inputted through `--with-projects` or `with-trace-fixtures`.

Reply:
I see, yeah forgot these were a bit coupled. Let's just get this working and can re-visit. If so I don't think we even need a flag.